tests/testStore.js

```diff
@@ -1,26 +0,0 @@
-import alt from '../src/alt/alt'
```

Comment:
They may be from the alt stuff, if so I can take a look

On Thu, Oct 8, 2015 at 2:19 PM Ramana Venkata <notifications@github.com>
wrote:

> In tests/testStore.js
> <https://github.com/Legitcode/tests/pull/15#discussion_r41570471>:
>
> > @@ -1,26 +0,0 @@
> > -import alt from '../src/alt/alt'
>
> Okay add it then. My PR got some cryptic errors. I'll try to fix them fast.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Legitcode/tests/pull/15/files#r41570471>.
>